frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature

```diff
@@ -14,35 +14,36 @@ Feature: Create Application from Devfile
             Given user has created workload "nodejs-ex-git" with resource type "Deployment"
               And user is at the Topology page
              When user right clicks on topology empty graph
-              And user selects "From Devfile" option from Add to Project context menu
+              And user selects "Import from Git" option from Add to Project context menu
               And user enters Git Repo url "https://github.com/redhat-developer/devfile-sample" in Devfile page
```

Comment:
```suggestion
              And user enters git repo url "https://github.com/redhat-developer/devfile-sample" in devfile form
```

Reply:
I've changed the `Devfile page` to `Import from Git` form but not touched `Git Repo` as Form has it in capitals